README.md

```diff
@@ -25,7 +26,7 @@ by adding `siftsciex` to your list of dependencies in `mix.exs`:
 ```elixir
 def deps do
   [
-    {:siftsciex, "~> 0.3.0"}
+    {:siftsciex, "~> 0.6.0"}
```

Comment:
I've reached out to the authors of this repo and they are not merging in any PRs and do not expect to add anything more to the repo, so I think in our version of the readme we should start pointing people specifically to our version by specifying the full github path. I've asked Sift about incorporating our fork into their SiftSciences github organization and start the process of moving the hexdocs to their repo, but no movement on that yet.

Reply:
Do you think it's worth to state this problem with the version in our README?

Reply:
I don't think we should be pointing to a version in our README, I think we should be pointing to our specific repo `master` branch for now until we can get it into the SiftSciences organization.

Reply:
btw i added this suggestion in my last commit